model/workflow_test.go

```diff
@@ -496,6 +496,9 @@ func TestTransitionUnmarshalJSON(t *testing.T) {
 }
 
 func TestDataInputSchemaUnmarshalJSON(t *testing.T) {
+
+	schemaName := FromString("schema name")
```

Comment:
Change the test to a json schema `Object`

Reply:
seems that it is not needed.